README.md

```diff
@@ -51,12 +51,10 @@ For more installation information see [installing on Windows](/docs/installation
 
 #### macOS
 
-To install the collector on macOS use the following brew commands:
+To install using the installation script, you may run:
 
 ```sh
-brew tap observiq/homebrew-observiq-otel-collector
-brew update
-brew install observiq/observiq-otel-collector/observiq-otel-collector
+sh -c "$(curl -fsSlL https://github.com/observiq/observiq-otel-collector/releases/latest/download/install_macos.sh)" install_macos.sh
```

Comment:
Could you explain real quick what this command does specifically, I've seen `sh` but I tried looking at the man pages for it and couldn't find the `-c` options I'm assuming it grabs the install file returned from that url posted and then installs it

Reply:
I'll send you a link over slack for it.